srcpkgs/ktouch/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'ktouch'
+pkgname=ktouch
+version=18.08.3
```

Comment:
18.12.0 has already been release, please update the template.

Reply:
@1is7ac3 ^